audit-ci.json

```diff
@@ -2,5 +2,5 @@
   "low": true,
   "package-manager": "auto",
   "registry": "https://registry.npmjs.org",
-  "allowlist": []
+  "allowlist": ["GHSA-9c47-m6qq-7p4h"]
```

Comment:
ignored as will be fixed by https://github.com/github/advisory-database/pull/1541 anyway